.storybook/preview.js

```diff
@@ -7,8 +7,8 @@
 
 import '@blueprintjs/core/lib/css/blueprint.css'
 import '@blueprintjs/datetime/lib/css/blueprint-datetime.css'
-import '@harness/design-system/dist/style.css'
-import '../src/styles/styles.css'
+import '../packages/design-system/src/index.tsx'
```

Comment:
Should this have changed from a css import to a tsx import?

Reply:
Earlier we were not using a sym-linked package, but now we are. So we will need to compile the css ourselves now.